pypdf/generic/_data_structures.py

```diff
@@ -969,20 +968,19 @@ def set_data(self, data: bytes) -> None:  # deprecated
 
 class ContentStream(DecodedStreamObject):
     """
-    In order to be fast, this datastructure can contain either:
-    * raw data in ._data
-    * parsed stream operations in ._operations
+    In order to be fast, this data structure can contain either raw data in ._data,
+    or parsed stream operations in ._operations.
```

Comment:
Why cannot we keep the list?

Reply:
It is reStructuredText.

Reply:
I probably should have known this.

I have reverted this to a list and am trying to change the following to be how it looks in pypdf.readthedocs.io:

> In order to be fast, this datastructure can contain either: * raw data in ._data * parsed stream operations in ._operations


> In order to be fast, this datastructure can contain either: 
>  * raw data in ._data 
>  * parsed stream operations in ._operations
>

Reply:
There is nothing wrong with asking - expecting that everyone knows about everything is just not useful ;). reStructuredText is indeed the default format for Sphinx, which can be observed by the `.rst` extensions inside the docs directory as well.